client/python/modeldb/__main__.py

```diff
@@ -0,0 +1,94 @@
+import dpath.util
+import os
+import json
+import sys
+
+def create_config():
+    file_dir = os.path.dirname(__file__)
+    filename = file_dir + '/syncer.json'
+    old_config = json.load(file(filename, 'r'))
+    new_config = json.load(file(filename, 'r'))
+
+    raw_input(
```

Comment:
I recommend changing this to a `print()` call. Pausing for input here gives the impression that there's a choice at this point and that pressing enter now will choose all defaults.